apis/dataplane/v1beta1/openstackdataplanenodeset_types.go

```diff
@@ -207,7 +207,7 @@ func (instance OpenStackDataPlaneNodeSet) GetAnsibleEESpec() AnsibleEESpec {
 var ContainerImageDefaults = openstackv1.ContainerImages{
 	ContainerTemplate: openstackv1.ContainerTemplate{
 		AgentImage:                    getStrPtr("quay.io/openstack-k8s-operators/openstack-baremetal-operator-agent:current-podified"),
-		AnsibleeeImage:                getStrPtr("quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest"),
+		AnsibleeeImage:                getStrPtr("quay.io/openstack-k8s-operators/openstack-ansibleee-runner:18.0.0-proposed-latest"),
```

Comment:
I don't think we can change it here and that too in the main branch.  It has to be updated in `OpenStackVersion` CR when deployed with 18.0.0-proposed branch I guess.

Reply:
but it updates the `OpenStackVersion`: https://logserver.rdoproject.org/01/1101/34cca5e4af767ffdd79db54ab79a415fa466e7f7/github-check/podified-multinode-edpm-deployment-crc/f80d6de/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackversions.core.openstack.org/controlplane.yaml